subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/plugin/kotlin/kotlintest/PluginFunctionalTest.kt.template

```diff
@@ -11,16 +12,19 @@ import kotlin.test.assertTrue
  * A simple functional test for the '${pluginId.value}' plugin.
  */
 class ${className.javaIdentifier} {
+    private val projectDir = Files.createTempDirectory("functionalTest").toFile()
+
+    private fun getBuildFile() = projectDir.resolve("build.gradle")
+    private fun getSettingsFile() = projectDir.resolve("settings.gradle")
+
```

Comment:
Ah, I was wrong that we were using JUnit Platform for Kotlin too :/

Could we use JUnit4's `TemporaryFolder` here? Something like:
```suggestion
    @get:Rule val tempFolder = TemporaryFolder()

    private fun getProjectDir() = tempFolder.root
    private fun getBuildFile() = getProjectDir().resolve("build.gradle")
    private fun getSettingsFile() = getProjectDir().resolve("settings.gradle")
```